ZeekPluginDynamic.cmake

```diff
@@ -164,7 +164,23 @@ if ( NOT ZEEK_PLUGIN_INTERNAL_BUILD )
         set(CMAKE_MODULE_PATH ${BRO_PLUGIN_BRO_CMAKE} ${CMAKE_MODULE_PATH})
 
         find_package(BinPAC REQUIRED)
-        find_package(CAF COMPONENTS core io openssl REQUIRED)
+
+        set(_saved_cmake_prefix_path "${CMAKE_PREFIX_PATH}")
+        list(INSERT CMAKE_PREFIX_PATH 0 "${CAF_ROOT_DIR}")
+        find_package(CAF REQUIRED COMPONENTS core io openssl CONFIG)
```

Comment:
Did you try simply passing `find_package(CAF REQUIRED COMPONENTS core io openssl CONFIG PATHS "${CAF_ROOT_DIR}")` instead of modifying `CMAKE_PREFIX_PATH`?

Reply:
Thanks for pointing out that option: no, I was a bit blindly following the suggestion of the error message from https://github.com/zeek/zeek/issues/1347.

After looking at the CMake docs, I still prefer setting `CMAKE_PREFIX_PATH` since the `PATHS` argument has the lowest search-priority and the typical, unmodified `CMAKE_PREFIX_PATH` has some common paths that theoretically risk picking up a CAF installation that doesn't match the one that Zeek reports building against.